### PR TITLE
made easier to derive entity from darkrp billboard

### PR DESCRIPTION
**Pull request by fruitwasp**

### CODE REVIEW

File: entities/entities/darkrp_billboard/cl_init.lua [hunk 1]
```diff
@@ -1,39 +1,55 @@
...
+    self.DrawPos = Vector(1, -111, 58)
+
+    self.Width = 558
+    self.Height = 290
+
+    self.HeaderMargin = 10
+    self.BodyMargin = 10
+
+    self.HeaderFont = "Trebuchet48"
+    self.HeaderFontSize = 48
```
Comment (Bo98) [inline entities/entities/darkrp_billboard/cl_init.lua]: Would `draw.GetFontHeight(self.HeaderFont)` work? (not tested)

Comment (fruitwasp) [inline entities/entities/darkrp_billboard/cl_init.lua]: didn't know about that one. nice one!

Comment (fruitwasp) [inline entities/entities/darkrp_billboard/cl_init.lua]: I case of an invalid font, it will spam errors though.

Comment (Bo98) [inline entities/entities/darkrp_billboard/cl_init.lua]: Does the drawing not do that anyway?

Comment (fruitwasp) [inline entities/entities/darkrp_billboard/cl_init.lua]: yah, true.

**Pull request merged**